### PR TITLE
resume doc basic interface

### PR DESCRIPTION
**Pull request by ATRJR**
added basic interface for resume doc, cannot be accessed unless you add link or whatever in index but im goted

### CODE REVIEW

File: student-hub-app/src/Resume/restumeDoc.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+          </div>
+        ))}
+        <input 
+          value={currentSkill} 
+          onChange={e => this.setState({ currentSkill: e.target.value })} 
+        />
+        <button onClick={this.addSkill} >Add Skill</button>
+      </div>
+    )
+  }
```
Comment (ryanmercado) [inline student-hub-app/src/Resume/restumeDoc.js:71]: This questionnaire ordering makes very little sense. In theory, it should first be General info, then skills, then job experience (with the ability to add multiple jobs) info, then projects (same deal) then extracurricular, and for each of these you will need to talk with Owen or reference the DB to see what fields are mandatory and what values each field needs

**Review by ryanmercado — CHANGES_REQUESTED** [review]
Also wth do you mean by it needs a link? 

As far as the resume questionnaire goes, we should probably make a questionnaire component for each of our 8 sections, starting with the mandatory ones, and then display them to the user one at a time, not allowing submit to be pressed unless required fields are filled.

Section order should probably be:

1. General Information
2. Skills List
3. Job Experience (1 job mandatory, optionally add more)
4. Project experience (1 mandatory...)
5. Extracurricular Activities List (each one will have a couple pieces ofdata)
6. Life Objective (fully optional)
7. Relevant Coursework List (fully optional)
8. Volunteer Work (fully optional)
9. Awards List (fully optional)


Will also probably need an interface for choosing to fill out optional fields

Will also need an interface for viewing each questionnaire portion again and editing it.

File: student-hub-app/src/Calendar/calendarLand.js
Comment (cdewitt02) [inline student-hub-app/src/Calendar/calendarLand.js:1]: this doesn't do anything why are u committing it it will lead to merge conflicts

**Review by cdewitt02 — CHANGES_REQUESTED** [review]

**Comment by Grayson34** [comment]
this pr should be taken down until way more work is done

**Pull request closed without merging**